### ISSUE TITLE
How does your project compare to IBus Typing Booster?

### ISSUE THREAD

**Issue by guettli** [bharadwaj-raju/TextSuggest#1]
How does your project compare to IBus Typing Booster?

https://fedoraproject.org/wiki/Features/Typing-Booster

I am interested in this topic since months. My current state of this is "wait until
I use Ubuntu 16.04" since it seems to have ibus-typing-booster on board.

Thank you for providing your code as open source

**Comment by bharadwaj-raju** [comment]
_Can_ be considered a re-implementation. In fact, I learned of the IBUS Typing Booster _after_ I made this, and I may not have done this project if I knew of it. :) 

Still, this project has the following advantages:
- custom words
- custom shortcuts/expansions
- lightweight
- works regardless of input method
- does not give suggestions until asked for, thus may reduce disturbance

Also, regarding "wait until I use Ubuntu 16.04", I myself use 16.04, and `ibus-typing-booster` is not in the repositories. It also seems that there is no PPA for it.
